etl-job/fhir_import_export.py

```diff
@@ -240,10 +242,25 @@ def _load_all(study: str,
                     limit=None, elastic_url=DEFAULT_ELASTIC,
                     output_path=None)
 
-        logs = fhir_put(project_id, path=file_path,
-                        elastic_url=DEFAULT_ELASTIC)
-        yaml.dump(logs, sys.stdout, default_flow_style=False)
-
+    # when generating graph with jsonschemagraph
+    except subprocess.CalledProcessError as exception:
```

Comment:
Guessing you've tested this manually already but It might be worth adding another end to end test in g3t to see if this new code is used.

Like for example, using some open access data that doesn't validate server side but validates client side and seeing if the expected error logs return as this PR has written them to.